swarm.go

```diff
@@ -201,11 +191,6 @@ func (s *Swarm) teardown() error {
 	return nil
 }
 
-// Process returns the Process of the swarm
-func (s *Swarm) Process() goprocess.Process {
-	return s.proc
```

Comment:
For posterity: this was very difficult to use correctly because _adding_ child processes raced with the process shutting down (leading to a panic if _this_ process closed before the child was added).

Technically, it could be used correctly with `AddChildNoWait`, but most callers would use `AddCHild` and `Go` (leading to panics on shutdown).